### PR TITLE
NO-JIRA: reduce node lease back off flames to 2

### PR DESCRIPTION
**Pull request by kannon92**

### CODE REVIEW

File: pkg/monitortests/node/kubeletlogcollector/monitortest.go [hunk 1]
```diff
@@ -71,8 +71,7 @@ func nodeFailedLeaseErrors(startedAt time.Time, finalIntervals monitorapi.Interv
 	const testName = "[sig-node] kubelet-log-collector detects node failed to lease events"
 	var failures []string
 
-	intervalsToFailOn := findLeaseIntervalsImportant(finalIntervals)
```
Comment (deads2k) [inline pkg/monitortests/node/kubeletlogcollector/monitortest.go:74]: need to filter this down to the type you're looking for, right?

File: pkg/monitortests/node/kubeletlogcollector/node.go [hunk 2]
```diff
@@ -526,49 +526,6 @@ func leaseFailBackOff(nodeLocator monitorapi.Locator, logLine string) monitorapi
...
-		// We will assume that intervals that have the same node and occur at the same time
-		// are duplicated events.
-		previousInterval := intervalsByNode[nodeName][len(intervalsByNode[nodeName])-1]
-		if val.From != previousInterval.From && val.To != previousInterval.To {
-			intervalsByNode[nodeName] = append(intervalsByNode[nodeName], val)
-		}
-	}
-	importantIntervals := monitorapi.Intervals(nil)
-	for _, val := range intervalsByNode {
-		if len(val) < 3 {
```
Comment (deads2k) [inline pkg/monitortests/node/kubeletlogcollector/node.go:557]: what if we reduce this to 2 so we flake if we get two instances and see how many hits we get?

Comment (kannon92) [reply pkg/monitortests/node/kubeletlogcollector/node.go:557]: updated.

**Comment by kannon92** [comment]
/retitle NO-JIRA: reduce node lease back off flames to 2

File: pkg/monitortests/node/kubeletlogcollector/node.go [hunk 3]
```diff
@@ -554,7 +554,7 @@ func findLeaseIntervalsImportant(intervals monitorapi.Intervals) monitorapi.Inte
 	}
 	importantIntervals := monitorapi.Intervals(nil)
 	for _, val := range intervalsByNode {
-		if len(val) < 3 {
+		if len(val) < 2 {
 			continue
 		}
 		for i, interval := range val {
```
Comment (deads2k) [inline pkg/monitortests/node/kubeletlogcollector/node.go:560]: in this for loop the `2`s need to become 1s

**Pull request merged**